### PR TITLE
Remove feed functions that are no longer used.

### PR DESCRIPTION
**Pull request by budzanowski**
As one of the comments in the deleted code mentions this should be
deprecated a long time ago. There is no way to trigger this code. Removing it will make working with the codebase easier.

### Testing 
1. No instructions really, the plugin should work as it did bc we have removed dead code. But please do a quick screen if it still works 🤷🏻

### CODE REVIEW

**Review by layoutd — APPROVED** [review]
The code looks OK, and in [smoke testing](https://github.com/woocommerce/facebook-for-woocommerce/wiki/Smoke-tests) I didn't see any issues. (I didn't run any of the automated tests)

I did add one comment regarding a change in the tests, and I also wonder whether the `public` methods should just be removed, or moved to the `Deprecated methods` section:
https://github.com/woocommerce/facebook-for-woocommerce/blob/fea56f60c00c25b666e1be5ab7da6ad8f6ae25b7/facebook-commerce.php#L3664-L3680

File: includes/test/facebook-integration-test.php [hunk 1]
```diff
@@ -75,97 +75,6 @@ public function __construct( $commerce ) {
 			);
 		}
 
-		/**
-		 * Test visible products by uploading feed.
-		 **/
-		function ajax_test_sync_products_using_feed() {
```
[the remark this answers is not in the document]
Comment (budzanowski) [reply includes/test/facebook-integration-test.php:81]: Removed in https://github.com/woocommerce/facebook-for-woocommerce/pull/2102/commits/149c9bc0da7875c4e7941e631285092ae9f1f063

**Comment by budzanowski** [comment]
>  I also wonder whether the public methods should just be removed, or moved to the Deprecated methods section

That is a good question. From my evaluation, this looks like a dead code so moving it to the deprecated section would not buy us much.  There may be some esoteric setups of the plugin that are using it but - I highly doubt it.

**Comment by budzanowski** [comment]
@layoutd thank you for the review!

**Pull request merged**